Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -88,7 +88,7 @@ struct FeatureFormExampleView: View {
                 .alert(
                     "The form wasn't submitted",
                     isPresented: model.alertIsPresented
-                ) {
+                ) { // swiftlint:disable:this vertical_parameter_alignment_on_call
```

Comment:
<img width="894" alt="Screenshot 2025-01-07 at 4 41 29 PM" src="https://github.com/user-attachments/assets/c0d120d0-931b-4638-a0cf-6a51a8e2a08f" />

This generates a new warning for me. I'm on Swiftlint 0.57.1.

Reply:
That didn't work. Since that comment is unrelated to these changes, I'll remove it, which then causes a warning for me. I'll have to look into that more.

Reply:
Somehow I had an older version of SwiftLint installed:

![image](https://github.com/user-attachments/assets/b6d6ba67-603b-406a-8678-00384551a8ce)

I don't know how it got there, but it isn't there anymore. Problem solved!